src/Translation/locales/es.json

```diff
@@ -67,17 +67,24 @@
     "connect": "Conexión",
     "connecting": "Conectando",
     "contributions": "Contribuciones",
+    "expired_at": "Tiempo expirado}",
```

Comment:
Whoops careful with the } here

Reply:
Ohhh 🤕 translation script. Will take a look

Reply:
Ooh something told me that it came from there :(